pixart.js

```diff
@@ -1 +1,30 @@
 
+$(document).ready(function canvas() {
+    for (i = 0; i<= 5000; i++)
+{
+
+  $( "body" ).append($("<div class='square'></div>"))
+}
+      this.squares = $(".square")
+});
+
+var inputColor = {
+    this.color = this.input.val()
+    return this.color
+  }
+
+$("button").on("click", function setColor(){
+   this.inputColor.css("background", this.color)
+ }
+
+ $("form").on("submit", function stop() {
+      stop.preventDefault()
```

Comment:
`preventDefault` is called on the event object. Take a look at these two sections of the Events & Callbacks lesson plan for more info...

* https://github.com/ga-wdi-lessons/js-events-callbacks#event-defaults
* https://github.com/ga-wdi-lessons/js-events-callbacks#the-event-object